subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/DefaultTestListener.kt

```diff
@@ -0,0 +1,23 @@
+package com.avito.android.build_verdict
+
+import org.gradle.api.tasks.testing.TestDescriptor
+import org.gradle.api.tasks.testing.TestListener
+import org.gradle.api.tasks.testing.TestResult
+
+abstract class DefaultTestListener : TestListener {
```

Comment:
Do you need this now, or better in separate PR about tests?

Reply:
How does this class complicate this PR?

Reply:
just the question of scope; like empty tests;
it's easy to change context after merge, decide about another implementation and forget about this class forever )

Reply:
It's already used so I don't get how I can forget about this ))